crates/walrus-core/src/encoding/quilt_encoding.rs

```diff
@@ -265,6 +505,130 @@ pub struct QuiltVersionV1;
 
 impl QuiltVersionV1 {
     const QUILT_VERSION_BYTE: u8 = 0x00;
+    const BLOB_HEADER_SIZE: usize = 7;
+    const EXTENSION_SIZE_BYTES_LENGTH: usize = 4;
+
+    /// Returns the total size of the encoded blob.
+    pub fn compute_encoded_blob_size(blob: &QuiltStoreBlob) -> Result<usize, QuiltError> {
```

Comment:
Remark: The naming is a bit confusing. Can we call this `serialized_blob_size` or something to avoid the confusion with blob encoding?